packages/yoroi-extension/app/api/ada/lib/state-fetch/types.js

```diff
@@ -340,3 +340,24 @@ export type TokenInfoResponse = {|
 |};
 export type PoolInfoFunc = (body: PoolInfoRequest) => Promise<PoolInfoResponse>;
 export type TokenInfoFunc = (body: TokenInfoRequest) => Promise<TokenInfoResponse>;
+
+export type CatalystRoundInfoRequest = {|
+  ...BackendNetworkInfo
+  |};
+
+export type CatalystRound = {|
+  +id: number,
+  +registrationStart: string,
+  +registrationEnd: string,
+  +votingStart: string,
+  +votingEnd: string,
+  +votingPowerThreshold: string,
+|};
+
+export type CatalystRoundInfoResponse = {|
```

Comment:
The API response has the `currentFund` and the `nextFund` but both of them are optional
if both of them are optional what should be the default. The app complaining this it needs a default values.